include/miopen/miopen.h

```diff
@@ -6587,6 +6587,9 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
  *  @{
  */
 
+#define ERROR_CODE_NEG_VALUE 1
```

Comment:
I think it's better to add ONEHOT_ prefix for both of these.

Reply:
I have added ONEHOT_ prefix to these macros.